engine/release-notes/index.md

```diff
@@ -23,6 +23,33 @@ for Docker Engine.
 
 # Version 20.10
 
+## 20.10.16
+2022-05-12
+
+This release of Docker Engine fixes a regression in the Docker CLI builds for
```

Comment:
I get the idea of doing a quick summary at the top, especially for a scenario of a "what's new" and dealing with, but this is a short list of items and info is repeated and close by below this paragraph. Just wondering it is necessary.

Reply:
Yeah, tried to stick with the format we used. I agree that for shorter change logs it's a bit redundant (although the summary can sometimes be written slightly more, well, as a summary, without the nitty-gritty details if people don't need them).